apc.php

```diff
@@ -807,7 +812,9 @@ function block_sort($array1, $array2)
 
 	$j = 0;
 	foreach (ini_get_all('apcu') as $k => $v) {
-		echo "<tr class=tr-$j><td class=td-0>",$k,"</td><td>",str_replace(',',',<br />',$v['local_value']),"</td></tr>\n";
+        //some settings (such as "apc.mmap_file_mask" or "apc.preload_path") may not have any values at all
+        //so PHP 8.1 generate error "PHP Deprecated:  str_replace(): Passing null to parameter #3 ($subject) of type array|string is deprecated"
+		echo "<tr class=tr-$j><td class=td-0>",$k,"</td><td>",(isset($v['local_value']) ? str_replace(',',',<br />',$v['local_value']) : ''),"</td></tr>\n";
```

Comment:
Maybe using `$v['local_value'] ?? ''` would be more elegant here? I don't think the comment is necessary.

Reply:
so it can't be used as package could be used on 7.0+

Reply:
@nikic pleae merge it as is

Reply:
Null coalesce was added in PHP 7.0, so I don't see the problem.

Reply:
Fixed in amended commit https://github.com/krakjoe/apcu/pull/444

Reply:
if the #444 will be merged this one should be closed